### ISSUE TITLE
Consider adding a `pk;delete` command for deleting proxied messages without reactions?

### ISSUE THREAD

**Issue by leocth** [PluralKit/PluralKit#472]
We recently had an issue where we can't delete our messages in a channel with reaction permissions disabled (for administrative purposes), and I would like a command to be added that deletes proxied messages without reactions. Something like `pk;delete [link]` would do, though I would also appreciate aliases like `pk;del` or `pk;x`. Thanks :smile_cat:

**Comment by rladenson** [comment]
This can already be done with `pk;message -delete <link|reply>`

**Comment by sisby-folk** [comment]
Functionality should be able to be near identical to `pk;edit` - where it can grab the most recent message or reply to a specific message, in place of copying the link/ID.

**Comment by rladenson** [comment]
You can already reply to a message with pk;msg

**Comment by leocth** [comment]
> This can already be done with `pk;message -delete <link|reply>`

Yeah but I think deleting a message should warrant its own command? Plus it's not actually documented anywhere in the command list currently (https://pluralkit.me/commands/#utility)

**Comment by rladenson** [comment]
At the very least it should definitely be documented yeah (along with the other undocumented pk;msg flag `-author`)

**Comment by leocth** [comment]
Still, I wonder if we can introduce `pk;delete` as an alias for `pk;message -delete` for increased visibility

**Comment by sisby-folk** [comment]
I think it would make sense being that high level (like pk;edit sets the precedent for) given that it *has* to be a reply to a message or include a link (and therefore can't be mixed up with `s delete` or `m delete`)

**Comment by rladenson** [comment]
(Added the docs update to the bugs notion board and the pk;delete suggestion to the [suggestions notion board](https://www.notion.so/74576534b6f9446a8ca9aa8c30c4069c?v=fd0532ef8542479aa02c8bc58154e314))

**Comment by Lilith-In-Starlight** [comment]
This would be a great feature to have. Also having it behave like pk;edit would be awesome too
